bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+
+Now you don't actually need to include the flag anymore.
+If this setting is turned on?
+
+Sjors Provoost: 00:18:44
+
+Yeah, I think that if you turn this setting on, then the flag does no longer have to be in there, it'll relate anyway so then I'm guessing there are other rules though because I assume the fee has to go up but I'm not sure about that
+
+Aaron van Wirdum: 00:18:56
+
+So in my case if so far I was running a Bitcoin Core node let's say what was the previous one 23 of course So far I was running Bitcoin Core 23 and then my node would not forward a transaction even if it had a higher fee and it conflicted with the previous transaction.
+It would not do that.
+But now with Bitcoin 24 I can switch the setting and now it will actually forward that transaction.
+
+Sjors Provoost: 00:19:22
+
+Yeah, I think that's it and it basically means that if you're relying on this opt-in RBF system to prevent double spends, basically to prevent double spends, you should now rely on that less so, because there's going to be more nodes that will relay this thing regardless.
+
+Aaron van Wirdum: 00:19:37
+
+Yeah, well, I mean, it was almost, well, I don't know about almost certainly, but there's definitely a much bigger chance that a conflict in a transaction will make it to a miner now, right?
+Because I think there only needs to be a relatively small amount of notes on the network that actually do it for a transaction to just find its way over the entire network.
+
+Sjors Provoost: 00:19:54
+
+But there were already ways to do it, right?
+There was a patch by Peter Todd which was pretty small and you could use that to modify your nodes.
+So there's already some nodes doing it.
+Now there's more, presumably.
+
+## Merchants
+
+Aaron van Wirdum: 00:21:06
+
+So does this mean that some merchants are going to be unhappy?
+
+
+Sjors Provoost: 00:21:12
+
+Well, they might be for that reason.
+
+
+Aaron van Wirdum: 00:21:13
+
+There are still some merchants that rely on zero-conf.
+
+Sjors Provoost: 00:21:15
+
+Yeah, there are definitely merchants that do that.
+But the flip side is that there are also merchants that use Lightning and Lightning is generally bothered a lot by this, by the existing RBF rules because they make it much more complicated to deal with penalty transactions.
+So they may get, in the long run, they may get a better lightning experience out of this.
+
+Aaron van Wirdum: 00:21:36
+
+I mean, my personal opinion is that this should just be the general rule.
+Transactions should always be broadcast and forwarded, especially if they have a higher fee.
+
+Sjors Provoost: 00:21:48
+
+I mean, maybe, we've already had this discussion during that episode, I guess, but I would also say that if you just turn on the RBF flag by default, it shouldn't bother you either.
```

Comment:
You'll notice that Sjors frequently uses 'I mean' as a filler phrase to start his sentences.